### DISCUSSION TITLE
Repositório com recursos acadêmicos

### DISCUSSION THREAD

**Discussion by emanoelvianna** [frontendbr/forum#1409] [Dicas]
Olá, tudo bem? 

Meu objetivo nesta _issues_ é compartilhar um [repositório](https://github.com/emanoelvianna/lista-interessante) que acabei criando durante a minha graduação. Na época eu sempre acabava indo além do conteúdo disponibilizado em aula, com isso achei interessante linkar esse conteúdo ao longo do tempo. Sendo assim, o repositório junta links redirecionando para professores de várias áreas do conhecimento. Na grande maioria você encontra recursos como material utilizado nas aulas ministradas, suas referências biográficas indicadas e outras informações.

Ele ainda não está tão organizado quanto eu gostaria, entretanto, espero que ele tenha algum conteúdo útil. Colaborações são bem vindas!

**Comment by flavianunes** [comment]
Os links dos professores 'Antonio Jr' e 'Marco A. Casanova' em banco de dados não funcionaram.

**Comment by flavianunes** [comment]
Material de banco de dados e Estrutura de Dados 2: http://www.decom.ufop.br/guilherme/
Prof. Dr. Guilherme Tavares de Assis

**Comment by emanoelvianna** [comment]
Oii, desculpa a demora em responder. Realmente deve existir alguns links que já não está mais ativos. Na verdade considerando minha falta de tempo atualmente está difícil verificar todos. 

Vou adicionar o material indicado!
